GitUI/CommandsDialogs/RevisionDiffControl.cs

```
@@ -825,7 +825,7 @@ private bool DeleteSelectedFiles()
                     bool isDir = (File.GetAttributes(path) & FileAttributes.Directory) == FileAttributes.Directory;
                     if (isDir)
                     {
-                        Directory.Delete(path, true);
+                        path.TryDeleteDirectory(out string _);
```

Fail silently?

---

This is an unknown submodule (as files are listed separately).
A messagebox could be added, but not important as this a special case. The directory will still be there...

---

The existing code appears to be failing silently. I wasn't convinced a popup was necessary here.
I propose we add it if we have a real usecase

---

IIUIC, it catched the exception and displayed it.

---

@mstv you're absolutely correct, I reverted this change